lib/plugins/features/movies/controller.js

```diff
@@ -8,6 +8,24 @@ exports.create = async (payload) => {
   Reflect.deleteProperty(payload, 'title');
 
   const movie = await new Movie().save(payload);
-
   return new Movie({ id: movie.id }).fetch();
 };
+
+exports.read = async (query) => {
+
+  return new Movie().query((qb) => {
+
+    if (query.year) {
+      // filter exact year
+      qb.where('release_year', '=', query.year);
+    } else if (query.from_year && query.to_year) {
+      // filter between 2 years
+      qb.where('release_year', '>=', query.from_year).andWhere('release_year', '<=', query.to_year);
+    } if (query.title) {
```

Comment:
nit: move this `if (query.title)` statement to a new line separated by whitespace.